CBI 896 show subscription details

evgeny-s opened the pull request.

naure commented on cere02/lib.rs:
```diff
@@ -840,9 +915,11 @@ mod ddc {
-            let bytes_ok = metrics.stored_bytes <= current_tier_limit[1];
-            requests_ok && bytes_ok
+            let bytes_ok = metrics.storage_bytes <= current_tier_limit.storage_bytes;
+            let wcu_ok = metrics.storage_bytes <= current_tier_limit.wcu;
```
There is some confusion here (my fault).

The limits expected by DDN are reserved capacity or throttling rates (e.g. wcu per minute). That is what the tier should mean right now.

On the other hand, the metrics would correspond to a limit per month, however that is not yet enforced or configurable (just measured).

In the interim, we can make both approaches consistent by extrapolating limits per minute to per month (i.e. simple multiplication). It means the app is allowed to use 100% reserved capacity all the time.

I suggest renaming the fields in tier and limits to `wcu_per_minute` (sync with Ulad).

evgeny-s replied:
Let's discuss it.
@naure

evgeny-s commented:
@naure , renamed parameters, based on what we discussed.
Also, removed this method: `is_within_limit` since it doesn't make sense anymore.
We don't use this method and it works incorrectly right n
